apps/architect-web/src/templates/behavioural-influence-networks.json

```
@@ -366,6 +366,11 @@
         "entity": "node",
         "type": "person"
       },
+      "behaviours": {
```

Biome (2.5.0)

[error] The key behaviours was already declared.

(lint/suspicious/noDuplicateObjectKeys)